usr/sbin/p11sak/p11sak.c

```diff
@@ -2550,6 +2775,49 @@ static CK_RV close_session(CK_SESSION_HANDLE session)
     return rc;
 }
 
+static CK_RV parse_file() {
+    CK_RV rc = CKR_OK;
+    FILE *fp;
+    char *file_loc = getenv("P11SAK_DEFAULT_CONF_FILE");
+
+    // open and parse file
+    if ((fp = fopen(file_loc, "r")) == NULL) {
```

Comment:
Calling fopen with `NULL` as path name is rather unusual.  Not sure if that works properly on all targets since I could not find a definition for this case in the spec of `fopen`.

Reply:
Done